zkstack_cli/crates/common/src/forge.rs

```diff
@@ -69,10 +70,17 @@ impl ForgeScript {
                 return Ok(res?);
             }
         }
-        let mut cmd = Cmd::new(cmd!(
-            shell,
-            "forge script {script_path} --legacy {args_no_resume...}"
-        ));
+
+        let command = format!(
+            "forge script {} --legacy {}",
+            script_path.to_str().unwrap(),
+            args_no_resume.join(" ")
+        );
+
+        // Displaying the command for debugging purposes
+        logger::debug(format!("Forge command: {}", command));
```

Comment:
@sanekmelnikov this is the only change that worries me a bit as it outputs the private key the script is called with. It is very helpful locally though, if a forge command fails, I can just copy and re-run the command separately without zkstack